cliparser/cliparser.go

```diff
@@ -91,15 +91,13 @@ func ParseCliArguments(args []string) (cliArguments CliArguments, err error) {
 		configurationPath = app.Flag("config", "A path to the configuration file").Short('c').String()
 		showProgress      = app.Flag("progress", "Show progress of stack creation. Option available only after setting up a remote sink").Bool()
 
-		onlineValidate                  = app.Command(ValidateMode, "Online template Validation")
-		onlineValidateTemplate          = onlineValidate.Arg("template", "A path to the template file.").Required().String()
-		onlineValidateLint              = onlineValidate.Flag("lint", "Enable template linting").Bool()
-		onlineValidateLintConfiguration = onlineValidate.Flag("lint-configuration", "A path to the configuration file").String()
-
-		offlineValidate                  = app.Command(OfflineValidateMode, "Offline Template Validation")
-		offlineValidateTemplate          = offlineValidate.Arg("template", "A path to the template file.").Required().String()
-		offlineValidateLint              = offlineValidate.Flag("lint", "Enable template linting").Bool()
-		offlineValidateLintConfiguration = offlineValidate.Flag("lint-configuration", "A path to the configuration file").String()
+		validate                  = app.Command(ValidateMode, "Online template Validation")
```

Comment:
Since we distinguish between online and offline validation no more I would delete `online` word from the description.